submissions/TaniaBondarenko/dom_task/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>dom_tb</title>
+<link rel="stylesheet" href="styles.css">
+<script src="my_javascript.js"></script>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Satisfy&display=swap"
+    rel="stylesheet">
+</head>
+<body>
+ <div class="container">
+    <header class="header_container">
+    <h2 class="head_title">P. I. Tchaikovsky "The Seasons"</h2>
+    </header>
+<div class=nav_content>
+    <div class="nav">
+    <button class="month" onclick="openMonth(event, 'jan' )">January: At the Fireside</button>
```

Comment:
It's not the optimal solution for adding click handlers. 
Let's make it more dynamic. I recommend using data-attributes for months shortcuts on each button and use event delegation for checking clicks on each button.
Avoid properties `onclick`. It's better to use `EventTarget.addEventListener()`

Reply:
fixed.
Thank you!